spec/queries/decidim/action_delegator/setting_delegations_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Decidim::ActionDelegator::SettingDelegations do
+  subject { described_class.new(setting) }
+
+  let(:organization) { create(:organization) }
+  let(:consultation) { create(:consultation, organization: organization) }
+  let(:granter) { create(:user, organization: organization) }
+  let(:grantee) { create(:user, organization: organization) }
+  let(:other_user) { create(:user, organization: organization) }
+
+  let!(:delegation) { create(:delegation, setting: setting, granter: granter, grantee: grantee) }
+  let!(:other_delegation) { create(:delegation, setting: setting, granter: granter, grantee: other_user) }
+  let!(:setting) { create(:setting, consultation: consultation) }
+  let!(:other_setting) { create(:setting) }
```

Comment:
I'd also create a delegation for this setting. The test could be passing because there are none for it.